specification/reservations/resource-manager/Microsoft.Capacity/stable/2020-10-25/quota.json

```diff
@@ -39,7 +39,7 @@
         "tags": [
           "QuotaInformation"
         ],
-        "description": "Get the current quotas (service limits) and usage of a resource. You can use the response from the GET operation to submit quota update requests.",
+        "description": "Get the current quota (service limits) and usage of a resource. You can use the response from the GET operation to submit quota update request.",
```

Comment:
(service limits) should be (service limit)

Reply:
Updated it.